src/modules/resolver.ts

```diff
@@ -20,6 +20,14 @@ export default class NoteResolver {
     return this.plugin.app.vault;
   }
 
+  private rename(file: TFile, newNotePath: string)
+  {
+    let old_file = this.vault.getAbstractFileByPath(file.path);
```

Comment:
this step should return the same TFile instance as the input file

Reply:
resolved